game/src/game/GameApp.cpp

```diff
@@ -24,16 +25,18 @@ using glm::ivec2;
 using std::make_unique;
 using std::string;
 
-GameApp::GameApp()
-{
-}
+using namespace std;
 
 /**
  * Runs once the windowing system has been initialized and the window
  * is created. Services should be added here.
  *
  * THIS IS CALLED BEFORE SERVICES ARE INITIALIZED
  */
+GameApp::GameApp()
+{
+}
+
```

Comment:
The comment above was supposed to be for GameApp::OnInit(), so can we move this constructor back above the comment?